### ISSUE TITLE
Getting the download url instead of downloading

### ISSUE THREAD

**Issue by alistvt** [instaloader/instaloader#1105]
Hello to everyone and thanks to the developers and all of the contributors to this great library.

Is it possible to instead of downloading posts (or stories, or whatever), just get their URL (URL of the photo on Instagram servers)?

**Comment by santinmike** [comment]
I think you can get it with the `.url` method of the `POST` class

**Comment by alistvt** [comment]
@santinmike no, this approach doesn't work if there are more than one picture in a post.

**Comment by anokhovd** [comment]
> @santinmike no, this approach doesn't work if there are more than one picture in a post.

Then i think you can use
https://github.com/instaloader/instaloader/blob/e67b9e6c37d9444fc8f5038d3ddbdfbe360599a4/instaloader/structures.py#L286
